addons/website_sale/models/res_config_settings.py

```diff
@@ -34,6 +34,7 @@ class ResConfigSettings(models.TransientModel):
                                                   related='website_id.cart_recovery_mail_template_id', readonly=False)
     cart_abandoned_delay = fields.Float("Abandoned Delay", help="Number of hours after which the cart is considered abandoned.",
                                         related='website_id.cart_abandoned_delay', readonly=False)
+    cart_add_on_page = fields.Boolean("Stay on page on addition to cart", related='website_id.cart_add_on_page', readonly=False)
```

Comment:
@xO-Tx Sorry, the wording wasn't great. Replace by "Stay on page after adding to cart"